README.md

```diff
@@ -80,42 +80,34 @@ docker run -d --name translate-all-in-one \
 
 ## TypeScript client
 
-### Installing
-
-#### npm
+### Dependencies
 
 ```bash
 npm config set @buf:registry  https://buf.build/gen/npm/v1/
 
-npm install @buf/expectdigital_translate-agent.bufbuild_connect-es@latest
-npm install @buf/expectdigital_translate-agent.bufbuild_es@latest
+npm install @connectrpc/connect
+npm install @connectrpc/connect-web  
+npm install @buf/expectdigital_translate-agent.connectrpc_es@latest
```

Comment:
Not required. These packages are already dependencies.

Commit package.json and pacakge-lock.json instead.

```suggestion
```

Reply:
@egijasp do you use yarn or npm? `yarn.lock` is part of the repo.

Reply:
Ignore my comment! I thought the change is in the webapp.

Reply:
at the start, i used yarn, then had problems installing dependencies for grpc client, so I had to use npm.